pkg/auth/authz_handler_test.go

```diff
@@ -463,6 +463,11 @@ var _ = Describe("authz", func() {
 			allowedRoles: []ocm.RoleType{ocm.AdminRole, ocm.ReadOnlyAdminRole, ocm.UserRole},
 			apiCall:      listComponentVersions,
 		},
+		{
+			name:         "list supported openshift versions",
```

Comment:
I don't think that a test here will help here, looks like this validations is not related directly the same code that run with swagger, so the permission here is mocked, if you set the permissions to be `allowedRoles` then this is how the mocks will be defined, if you want to validate the change in the swagger, i needs to be done in the subsystem.
@rollandf correct me if i'm wrong.

Reply:
The test fails if run before the swagger change, so it looks it does it OK.

Reply:
The test fails if run before the swagger change, so it looks it does it OK.

Reply:
it compare to the validation in the swagger? because i see that it mock the capability `passCapabilityReview`

Reply:
ok i see it. thanks